src/main.rs

```diff
@@ -25,6 +26,7 @@ fn main() {
         (@subcommand init =>
             (about: "Create a new hermit shell")
             (usage: "hermit init [shell-name]")
+            (@arg INPUT: "name of new shell-name")
```

Comment:
help text should probably just be "name of new shell".  Also, thanks for using the new terminology!!